test/e2e/utils/bootstrap.ts

```diff
@@ -0,0 +1,39 @@
+import * as http from 'http';
+import { bootstrapMicroframework, Microframework } from 'microframework-w3tec';
+import { Application } from 'express';
+import { Connection } from 'typeorm/connection/Connection';
+import { expressLoader } from './../../../src/loaders/expressLoader';
+import { winstonLoader } from './../../../src/loaders/winstonLoader';
+import { swaggerLoader } from './../../../src/loaders/swaggerLoader';
+import { monitorLoader } from './../../../src/loaders/monitorLoader';
+import { homeLoader } from './../../../src/loaders/homeLoader';
+import { typeormLoader } from '../utils/typeormLoader';
+import { publicLoader } from './../../../src/loaders/publicLoader';
```

Comment:
I think it's unused. Please fix your tslint or ts => https://palantir.github.io/tslint/rules/no-unused-variable/

Reply:
Please check the whole file